include/openssl/ocsp.h.in

```diff
@@ -60,6 +60,12 @@ use OpenSSL::stackhash qw(generate_stack_macros);
 
 # ifndef OPENSSL_NO_OCSP
 
+# ifndef STACK_OF_OCSP_RESPONSE
+/* Protect against recursion */
```

Comment:
Why would this be needed? The header file is protected against recursion as a whole.

Reply:
true, this code exist also for #20945, it is fixed here now